policy/resolver_test.go

```diff
@@ -132,19 +132,28 @@ policies:
 		{asset: "asset1", policies: []string{policyMrn("policy1")}},
 	}, []*policy.Bundle{b})
 
-	checkResolvedPolicy := func(t *testing.T, rp *policy.ResolvedPolicy) {
-		require.Len(t, rp.ExecutionJob.Queries, 3)
-		require.Len(t, rp.Filters, 1)
-	}
-
 	t.Run("resolve with correct filters", func(t *testing.T) {
 		rp, err := srv.Resolve(context.Background(), &policy.ResolveReq{
 			PolicyMrn:    policyMrn("policy1"),
 			AssetFilters: []*explorer.Mquery{{Mql: "true"}},
 		})
 		require.NoError(t, err)
 		require.NotNil(t, rp)
-		checkResolvedPolicy(t, rp)
+		require.Len(t, rp.ExecutionJob.Queries, 3)
+		require.Len(t, rp.Filters, 1)
+		require.Len(t, rp.CollectorJob.ReportingJobs, 3)
+
+		qrIdToRj := map[string]*policy.ReportingJob{}
+		for _, rj := range rp.CollectorJob.ReportingJobs {
+			qrIdToRj[rj.QrId] = rj
+		}
+		// scoring queries report by code id
+		require.NotNil(t, qrIdToRj[b.Queries[1].CodeId])
+		// data queries report by mrn
```

Comment:
Is `codeId` vs. `mrn` by intention, or should we clean this up in a follow-up?

Reply:
currently by intention. scoring queries go by code id, data queries now go by mrn (this is new, we havent had reporting jobs for data queries before)

in the future, i think we want everything to report by mrn